docs/sources/datasources/azuremonitor.md

```diff
@@ -302,128 +287,15 @@ If you're not currently logged in to the Azure Portal, then the link opens the l
 
 <div class="clearfix"></div>
 
-### Logs alerting
-
-> Only available in Grafana v7.0+.
-
-Grafana alerting is supported for Application Insights. This is not Azure Alerts support. Read more about how alerting in Grafana works in [Alerting rules]({{< relref "../alerting/_index.md" >}}).
-
-## Query Application Insights service
```

Comment:
Application Insights and Insights analytics have been deprecated and cannot be used in new panels